source/whats-new.txt

```diff
@@ -32,6 +34,26 @@ following versions of the {+php-library+}:
 
 * :ref:`Version 1.15 <php-lib-version-1.15>`
 
+.. _php-lib-version-2.0:
+
+What's New in 2.0
+-----------------
+
+The {+library-short+} v2.0 release includes the following features,
+improvements, and fixes:
```

Comment:
```suggestion
The {+library-short+} v2.0 release includes the following features,
improvements, and fixes:
```
Question: I like this introductory line. Should we start including at the beginning of each What's New entry from now on for consistency?

Reply:
I can add